anta/tests/routing/bgp.py

```diff
@@ -323,6 +327,10 @@ def test(self) -> None:
 
             afi = cast(Afi, command.params.get("afi"))
             safi = cast(Optional[Safi], command.params.get("safi"))
+
+            # Swaping AFI and SAFI in case of SR-TE address family
+            if afi == "sr-te":
+                afi, safi = safi, afi
```

Comment:
I think we should put that in the `render` function no?

Reply:
yes we are doing this in render function too in the line (https://github.com/arista-netdevops-community/anta/pull/566/files#:~:text=%23%20For%20SR%2DTE,.vrf)))

Reason of here again to change is to get correct afi and safi in failure. If we dont swap then for SR-TE in failure we get afi as sr-te but expected is ipv4 and safi as sr-te.
```
Failures: [{'afi': 'sr-te', 'safi': 'ipv4', 'vrfs': {'default': {'10.255.0.2': {'peerState': 'Connect', 'inMsgQueue': 0, 'outMsgQueue': 0}}}}]
```